client/js/screens/Signup/styles.js

```diff
@@ -0,0 +1,41 @@
+import {StyleSheet} from 'react-native';
+import {typography} from '../../config/styles';
+
+const styles = StyleSheet.create({
+  pageContainer: {
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    // height: '550%',
+  },
+  signinHeroImage: {
+    resizeMode: 'contain',
+    height: 150,
+    marginTop: 50,
+    marginBottom: 50,
```

Comment:
use flexbox properties, not huge margins.

Reply:
Flexed the parent container to JustifyContent 'center' but kept the 50 margin button to add space between the form and image.